utils/rpms/daos.spec

```diff
@@ -358,6 +358,9 @@ getent group daos_admins >/dev/null || groupadd -r daos_admins
 %{_libdir}/*.a
 
 %changelog
+* Fri June 05 2020 Ryon Jensen <ryon.jensen@intel.com> - 1.1.0-20
```

Comment:
still missing the bump of "Release" at the top"

Reply:
Got it this time.